lib/RemoveVariableEventBehaviour.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var inherits = require('inherits');
+
+var CommandInterceptor = require('diagram-js/lib/command/CommandInterceptor').default;
+var is = require('bpmn-js/lib/util/ModelUtil').is;
+var getBusinessObject = require('bpmn-js/lib/util/ModelUtil').getBusinessObject;
+
+/**
+ * Remove 'camunda:variableEvents' property when a startEvent is moved out of an event subProcess
+ */
+function RemoveVariableEventBehaviour(
+    modeling, injector, elementRegistry
+) {
+
+  injector.invoke(CommandInterceptor, this);
+
+  this.postExecuted(['shape.move'], function(context) {
+
+    var oldParent = context.oldParent;
+    var oldParentBusinessObject = getBusinessObject(oldParent);
+    var newParent = context.newParent;
+    var shape = context.shape;
+    var shapeBusinessObject = getBusinessObject(shape);
+    var eventDefinitions, definitionsCopy;
+
+    // if shape is a startEvent
+    if (is(shape, 'bpmn:StartEvent')) {
+
+      eventDefinitions = shapeBusinessObject.get('eventDefinitions');
+      definitionsCopy = eventDefinitions.slice();
+
+      definitionsCopy.forEach(def => {
+
+        // if event definitions have variableEvents property
+        if (def.get('camunda:variableEvents')) {
+
+          // if shape within event subpprocess changes parent
+          if (is(oldParent, 'bpmn:SubProcess') && !is(newParent, 'bpmn:SubProcess') && oldParentBusinessObject.get('triggeredByEvent')) {
```

Comment:
The order of the conditions could be confusing (oldParent, newParent, oldParent...). Maybe we can do something like

```js
 if (
  is(oldParent, 'bpmn:SubProcess') && 
  oldParentBusinessObject.get('triggeredByEvent') && 
  !is(newParent, 'bpmn:SubProcess')
) { 
  ... 
}
 ```